changelog/9742.improvement.rst

```diff
@@ -1 +0,0 @@
-Display assertion message without escaped newline characters with ``-vv``.
```

Comment:
This seems to have been removed by accident?

Reply:
Sorry, I thought that file is dropped there as an example. I'll bring it back.